dao/src/main/kotlin/repositories/analyzerrun/AnalyzerRunsTable.kt

```diff
@@ -63,12 +68,12 @@ object AnalyzerRunsTable : LongIdTable("analyzer_runs") {
     private fun loadAnalyzerRun(resultRow: ResultRow): AnalyzerRun {
         val analyzerRunId = resultRow[id].value
         val analyzerJobId = resultRow[analyzerJobId].value
-        val environment = checkNotNull(EnvironmentsTable.getById(resultRow[environmentId].value))
-        val config = checkNotNull(AnalyzerConfigurationsTable.getByAnalyzerRunId(analyzerRunId))
-        val projects = ProjectsAnalyzerRunsTable.getProjectsByAnalyzerRunId(analyzerRunId)
-        val packages = PackagesAnalyzerRunsTable.getPackagesByAnalyzerRunId(analyzerRunId)
-        val ortRunId = checkNotNull(AnalyzerJobsTable.getOrtRunIdById(analyzerJobId))
-        val issues = OrtRunsIssuesTable.getIssuesByOrtRunId(ortRunId, AnalyzerRunDao.ISSUE_WORKER_TYPE)
+        val environment = checkNotNull(GetEnvironment(resultRow[environmentId].value).execute())
```

Comment:
I would it find slightly more readable if the query classes ended on `Query`, e.g. `GetEnvironmentQuery(...).execute()` rather than `GetEnvironment(...).execute()`. But maybe this is just personal taste.